### PR TITLE
Section 3

### PR DESCRIPTION
**Pull request by CalvinRodo**
Added section 3 with basic validations.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -78,4 +78,8 @@ credentials
 
 
 
-public/js/dist
\ No newline at end of file
+public/js/dist
+
+# Will uncomment when we are setting up langauge keys
```
Comment (jeffmaher) [inline .gitignore]: Can remove this line

```suggestion
# Will uncomment when we are setting up langauge keys
```

**Review by jeffmaher — DISMISSED** [review]
LGTM! Tested in Chrome and just one little nitpick in the .gitignore file.

**Review by dsamojlenko — DISMISSED** [review]
🌮

File: .gitignore (hunk 1 above)
Comment (CalvinRodo) [reply .gitignore]: Yes I'll also remove the changes to the .gitignore for now so we either stop tracking the files i.e. delete them or we just check in all of the language strings

**Review by dsamojlenko — APPROVED** [review]
Looks good!

**Pull request merged**